### DISCUSSION TITLE
改进IOI联机

### DISCUSSION THREAD

**Discussion by uB1nlR** [Meloong-Git/PCL#268] [提问] [answered]
HiPer，可能有些人没有1元钱的资本（悲）
而且是每次使用都需要花费积分，若有玩时间长的话确实不很实惠，尤其对于学生党
若可以的话，希望可以改进一下IOI的联机！
龙猫大大辛苦了！

**Comment by uB1nlR** [comment]
az
好吧
虽然还是希望白嫖（bushi）

**Comment by zkitefly** [comment]
目前 IOI 模块已经停更，现在的主线是 HiPer 模块
IOI 应该是不会更新了的，猫猫那边说已经不更新了的

你可以去加入群里，看看有没有人给你帮你白嫖索引码qwq

**Comment by ffip** [comment, marked as answer]
有没有一种可能iOi我（我是iOi和HiPer的作者）源码都没有了
且当时屎山(具体参照当年更新）到不想动了所以才有了后来的Cato和HiPer。【如果我没记错，有一期我们的更新是 我们解决了闪断的问题，它再也不会出现了，因为现在根本就连不上了】
现在想想当时的离谱情况，即使我源码还在也真的不想去折磨自己了。当然iOi的表现差这跟国内的网络大环境也有很大关系，iOi在海外国际网络环境下表现还是很出彩的。另外是iOi除了我开发成本以外几乎无需成本，但HiPer需要强大的网络资源保障其网络服务质量与社群运维，希望得到理解和支持，未来我们在情况允许的情况下调整资费，但至少目前还做不到。
至于费用问题，我能理解，好像时间越长越便宜。
除此之外，就看你愿不愿意付出时间在HiPer社群建设方面。
如果有付出，喵点应该是不缺的。
另外储值方面是有很多活动和优惠的。

目前具体的HiPer定价和活动奖励范围几率如下/：

喵点:
๑ 1元人民币10喵点
  (不含赠送/活动/奖励)

定价:
๑ 3小时:
   普通用户 = 5 喵点
   ◍每日一次特价优惠◍
   ◍第二次将变更为24小时◍
๑ 24小时:
   会员用户 = 10 喵点
   普通用户 = 20 喵点
๑ 30天:
   会员用户 = 225 喵点
   普通用户 = 300 喵点
๑ 90天: 
   会员用户 = 625 喵点
   普通用户 = 800 喵点
๑ 360天:
   会员用户 = 1780 喵点
   普通用户 = 2200 喵点
๑ 会员:
   首次加入 = 300 喵点
   每月续展 = 200 喵点

概率:
๑ 十分好运:
   0-6 15% 7-14 60% 15-30 25%
๑ 每日打卡:
   0-6 15% 7-14 60% 15-30 25%
๑ 幸运共享:
   0-6 15% 7-14 60% 15-30 25%
๑ 奇遇抽奖:
   订单池抽奖，每周日开奖。
   多次参与可叠加命中几率，
   直至用户中奖后几率重置。
๑ 储值随机:
   按单次储值金额进行奖励范围1%~100%

我能理解现在学生手里紧张，但是紧张到连十多元都拿不出来的学生，
我建议玩游戏的时间多去读书或者兼职改变生活会比玩游戏收获来的更多。

**Reply by uB1nlR** [reply]
嗯嗯好的
感谢你们的付出！

**Reply by ffip** [reply]
更快的速度 更稳的网络 更好的服务 更低的价格 始终是我们努力的方向和目标，如果情况允许的话，请以力所能及的方式支持我们吧。祝您游戏愉快，感谢您对PCL及iOi、HiPer的厚爱。

**Reply by z0z0r4** [reply]
GitHub上不应该涉及讨论金钱交易，何况是抽奖
就算是你列出的"优惠"
